assets/src/components/SidebarFields/TextSidebarField.js

```diff
@@ -1,9 +1,10 @@
 import { TextControl } from '@wordpress/components';
 import { __ } from '@wordpress/i18n';
 
-export const TextSidebarField = ({ value, setValue, label }) => (
+export const TextSidebarField = ({ value, setValue, label, type }) => (
   <TextControl
     label={label}
+    type={type}
```

Comment:
This doesn't take care of any validations, so you you're able to enter any string here and it doesn't check if it's a valid URL 😕 for this we actually already have a [component](https://github.com/greenpeace/planet4-plugin-gutenberg-blocks/blob/main/assets/src/components/URLInput/URLInput.js) so maybe we could use it here?